Month1/Week2/Section1/Exercise1/main1.cpp

```diff
@@ -0,0 +1,42 @@
+/*Exercise 1 - Understanding classes
+Create a class called FirstClass with a public constructor and destructor. Have the constructor output �I�m in the constructor!� and have the destructor output �I�m in the destructor!�. Then, instantiate 5 classes and run your program, are the constructor and destructor calls where you expected them to be?
+
+Now create a loop of at least five iterations. For each iteration, instantiate your class once. Run your program, are the constructor and destructor calls where you expected them to be?
+
+*/
+#include <iostream>
+
+#include<string>
+
+class FirstClass
+{
+public:
+	std::string Texto;
+	FirstClass()
+	{
+
+	}
+	FirstClass(std::string texto)
```

Comment:
Remember all variables and functions should use PascalCase. `texto` should be `Texto` (Though remember to also fix the name to make it English).

For conflicts between member names and constructor parameter names, it's acceptable to use the prefix `In` if you want. So you could call this `InText`, for example.

Reply:
However, for this exercise, you should only have a default constructor and a default destructor, each with their appropriate text printed to the string. The `Texto` member and the `Print` member should not be included in the class, at it defeats the purpose of the exercise.